_layouts/event.html

```diff
@@ -1,9 +1,10 @@
 ---
 layout: page
 ---
-<div class="header-card">
+<div class="header-card calendar-button">
```

Comment:
This `div` is not a `calendar-button`. Naming this one seems a little tricky, `calendar-button-container` maybe? Though looking at they styling a `header-card-with-calendar-button` may be better? Not 100% sure so open to alternatives